regression/cbmc/Array_operations3/main.c

```diff
@@ -0,0 +1,5 @@
+int main()
+{
+  void *p;
+  __CPROVER_array_set(p, 0);
```

Comment:
I hope that nobody will read this test as a promise of some sort of functionality.

Reply:
We should probably emit some sort of verification condition.

Reply:
Done.